src/utils/constants.js

```diff
@@ -0,0 +1,5 @@
+module.exports.FETCH_POSTS_PER_TAG = 2;
+module.exports.HOT = 'hot';
+module.exports.TRENDING = 'trending';
+module.exports.GET_DISCUSSIONS_BY_HOT = 'get_discussions_by_hot';
+module.exports.GET_DISCUSSIONS_BY_TRENDING = 'get_discussions_by_trending';
```

Comment:
Yeah exactly what you did here, storing values in variables, exporting them, importing them where needed and using the var names to compare - do this with the `determinePostType` method

Reply:
Soudsn good.  Added some constants for the postType field in posts.